crates/bevy_window/src/lib.rs

```diff
@@ -33,6 +35,11 @@ impl Default for WindowPlugin {
     }
 }
 
+#[derive(Default)]
```

Comment:
It's also worth `#[derive(Debug, Clone)]` as well, since these are usually pretty fundamental derives that are always useful.

Reply:
Great, added! :) (my Rust noobiness showing)